tests/test_cases/issue_120/issue_120.py

```diff
@@ -23,7 +23,7 @@ def monitor(dut):
         raise TestFailure("stream_in_valid should be high on the 5th cycle")
 
 
-@cocotb.test()
+@cocotb.test(expect_error=cocotb.triggers.TriggerException if cocotb.SIM_NAME.startswith(("xmsim", "ncsim")) and cocotb.LANGUAGE in ["vhdl"] else False)
```

Comment:
Do you know exactly which line this occurs on? It might be better to catch it in place.

Reply:
It's in https://github.com/cocotb/cocotb/blob/05d49c0dbe481712443ec7b7517a4fb9686f5943/tests/test_cases/issue_120/issue_120.py#L20